jira.py

```diff
@@ -0,0 +1,27 @@
+import os
+import sys
+
+from lib.jira_conn import JiraAPIClient
+
+
+# JQL query All QA Requests since 2022 filter_id: 13856
+query = 'jql=filter=13856&fields=id,key,status,created,\
```

Comment:
pass the filter id as a parameter.
Also explain the customfield, add them as variable.